peachjam/templates/peachjam/_case_histories.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n %}
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>{% trans 'Case' %}</th>
+      <th>{% trans 'Outcome' %}</th>
+      <th>{% trans 'Court' %}</th>
+      <th>{% trans 'Judges' %}</th>
+      <th>{% trans 'Date' %}</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for history in document.case_histories.all %}
```

Comment:
It will actually be better to load this all in the view, so that the right fields are prefetched. Otherwise you're going to the DB for every history, and for every historical_judgment, and for every judge, which could be 5-10 queries.

Reply:
done